frontend/src/pages/Transactions.vue

```diff
@@ -50,12 +51,22 @@ export default {
   data() {
     return {
       timestamp: Date.now(),
+      titel_gdt: this.$t('gdt.gdt'),
     }
   },
   methods: {
     updateTransactions(pagination) {
       this.$emit('update-transactions', pagination)
     },
+    showGdtBalance(Boolean) {
```

Comment:
```suggestion
    showGdtBalance(boolean) {
```

Reply:
No capital letter for variables. This looks like a class and not like a variable